airflow/contrib/operators/dataflow_operator.py

```diff
@@ -182,7 +195,8 @@ def execute(self, context):
         dataflow_options.update(self.options)
 
         hook.start_java_dataflow(self.job_name, dataflow_options,
-                                 self.jar, self.job_class)
+                                 self.jar, self.job_class, self.add_random_jobname_suffix,
```

Comment:
is "add_random_jobname_suffix"  same as "append_job_name"?
if so, why not use "append_job_name" instead to keep it in sync with func arg name.

Reply:
> is "add_random_jobname_suffix" same as "append_job_name"?


yes

> if so, why not use "append_job_name" instead to keep it in sync with func arg name.

I changed it in order to make it clearer as to what it does since it would be user facing and might cause confusion. 
Let me know what you think on this.

Reply:
What is the sample string appended by "append_job_name"?

Reply:
I have added an example of each job name in the docstring now. "append_job_name" adds a random hash string like "eccd9700"